src/Features/Core/Portable/Common/SymbolDisplayPartKindTags.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis
+{
+    internal static class SymbolDisplayPartKindTags
+    {
+        public static string GetTag(SymbolDisplayPartKind kind)
+        {
+            switch (kind)
+            {
+                case SymbolDisplayPartKind.AliasName:
+                    return TextTags.Alias;
```

Comment:
Nit.  But why don't these names match?  Why have AliasName in one of our APIs and Alias in another?  it seems like we'd want these to be consistent.